graphql.schema.json

```diff
@@ -12456,6 +12472,12 @@
             "isDeprecated": false,
             "deprecationReason": null
           },
+          {
```

Comment:
It looks like there are some AHA-related changes that don't really belong with this diff(?) I think they may be already on release-173, so maybe all this needs is to have the base branch merged in?